parachain/pallets/ethereum-beacon-light-client/src/lib.rs

```diff
@@ -0,0 +1,666 @@
+//! # Ethereum 2 Light Client Verifier
+//!
+//! This module implements the `Verifier` interface. Other modules should reference
+//! this module using the `Verifier` type and perform verification using `Verifier::verify`.
+#![allow(unused_variables)]
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(test)]
+mod mock;
+
+#[cfg(test)]
+mod tests;
+mod merklization;
+
+use codec::{Decode, Encode};
+use frame_support::{dispatch::DispatchResult, log, transactional, Deserialize, Serialize};
```

Comment:
The `Deserialize`, `Serialize` traits are only available with `std` enabled.

```suggestion
use frame_support::{dispatch::DispatchResult, log, transactional};
```